backend/src/main/kotlin/com/example/demo/service/CardService.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.demo.service
+
+import com.example.demo.controller.dto.CreateCardRequest
+import com.example.demo.controller.dto.CardDTO
+import com.example.demo.model.Card
+import com.example.demo.model.User
+import com.example.demo.model.Ownership
+import com.example.demo.repo.CardRepository
+import com.example.demo.repo.OwnershipRepository
+import com.example.demo.repo.UserRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.data.domain.Pageable
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import java.util.*
+import java.util.stream.Collectors
+
+@Service
+class CardService(@Autowired private val cardRepository: CardRepository,
+                  @Autowired private val ownershipRepository: OwnershipRepository) {
+
+    public fun create(card: Card): Card {
```

Comment:
DB model should not be exposed to clients. 
You should have a class dto which is the one you expose different that Card (Card should be named CardEntity)